File: POO/excepciones/JavaVersion/src/com/Operador.java
package com;

public class Operador {
    // Atributos
    private String signo = "";

    // Constructor
    public Operador(String pOperador){
        this.signo = pOperador;
    }

    //Methos
    public String getSigno() {
        return signo;
    }
    public void setSigno(String signo) {
        this.signo = signo;
    }

    @Override
    public String toString() {
        String text = "";
        text += this.getSigno();
        return text;
    }
}

File: POO/Ejercicio 3/PythonVersion/com/Operador.py
class Operador:
    # Atributos
    signo = str
    
    # Constructor
    def __init__(self, pOperador) -> None:
        self.signo = pOperador
        
    # Métodos
    def get_signo(self) -> str:
        return self.signo
    def set_signo(self, pSigno):
        self.signo = pSigno
        
    def to_string(self) -> str:
        text = str
        text += self.get_signo()
        return text